TgeeT/Questioner
# Feature  #163046714 Add upcoming meetup endpoint

#### What does this PR do?

Create an endpoint for showing the upcoming meetups
#### Description of Task to be completed?

write failing test, 
write the bear minimum code to pass the tests

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
3. git checkout to ft-upcoming-meetup-163046714 branch
6. flask run the app and use postman to verify via *api/v1//meetup/upcoming* url

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2235327
#163046714

### screenshots
###### with no upcoming meetup
![image](https://user-images.githubusercontent.com/45405036/51188800-b84fd200-18ef-11e9-870d-870271f0d42e.png)

###### upcoming meetup
![image](https://user-images.githubusercontent.com/45405036/51188928-fe0c9a80-18ef-11e9-9a71-f055d250c67b.png)
